config/nightwatch.php

```diff
@@ -6,6 +6,8 @@
     'deployment' => env('NIGHTWATCH_DEPLOY'),
     'server' => env('NIGHTWATCH_SERVER', (string) gethostname()),
     'capture_exception_source_code' => env('NIGHTWATCH_CAPTURE_EXCEPTION_SOURCE_CODE', true),
+    'capture_request_body' => env('NIGHTWATCH_CAPTURE_REQUEST_BODY', false),
+    'redact_keys' => explode(',', env('NIGHTWATCH_REDACT_KEYS', 'password,password_confirmation')),
```

Comment:
I worry this might imply we'll redact these keys _anywhere_ in Nightwatch, and while that would be nice if we could, it won't always be possible.

Also technically it redacts the values, not the keys...